test/test_page/indexNode.js

```diff
@@ -0,0 +1,7 @@
+var xcomponentapi = require("../../lib/xcomponentapi.node.js");
+
+var serverUrl = "wss://localhost:443";
+
+xcomponentapi.default.getXcApiList(serverUrl, function (connection, apis) {
```

Comment:
why xcomponentapi.default ?
We'd like something similar to the browser api style: xcomponentapi.getXcApiList() without default